### PR TITLE
fix: text component docs

### PR DESCRIPTION
**Pull request by codepretty**
# Updating the text doc to have all examples & have accessibility info
- all text types, states, and variations now have shorthand & children examples
- fixed some cases where children/shorthand were reversed

**Before**
image

**After**
image

### CODE REVIEW

**Review by miroslavstastny — APPROVED** [review]

File: docs/src/examples/components/Text/Types/TextSizesExample.shorthand.tsx [hunk 1]
```diff
@@ -3,21 +3,21 @@ import { Text } from '@stardust-ui/react'
 
 const TextSizesExampleShorthand = () => (
   <div>
-    <Text size="xs">Dicta voluptatum dolorem.</Text>
+    <Text size="xs" content="Dicta voluptatum dolorem." />
```
Comment (kuzhelov) [inline docs/src/examples/components/Text/Types/TextSizesExample.shorthand.tsx:6]: 👍

File: docs/src/examples/components/Text/Types/TextSizesExample.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Text } from '@stardust-ui/react'
+
+const TextSizesExample = () => (
+  <div>
+    <Text size="xs">Dicta voluptatum dolorem.</Text>
```
Comment (kuzhelov) [inline docs/src/examples/components/Text/Types/TextSizesExample.tsx:6]: actually, this raises another point related to the values used for sizes. The problem is that it is hard to intuitively reason for the client about, say, which one is bigger: `xl` or `2x`. Would propose to use values that are more intuitive in this sense (and address the issue by means of separate PR). 

Quite similar sentiments are expressed in #136.

**Review by kuzhelov — CHANGES_REQUESTED** [review]
`changelog.md` should be updated as this PR introduces publicly visible changes

**Review by kuzhelov — APPROVED** [review]

File: docs/src/examples/components/Text/Types/TextSizesExample.tsx (hunk 2 above)
Comment (codepretty) [reply docs/src/examples/components/Text/Types/TextSizesExample.tsx:6]: Completely agree. My next PR is going to use a theme-based enum to show the appropriate sizes. 

What I'm still not sure about is what to do for naming and how that could vary among themes...

**Pull request merged**